memcached/README.md

```diff
@@ -0,0 +1,97 @@
+<!--
+Copyright (c) 2014-2015 Google Inc. All rights reserved.
```

Comment:
You're right, this was not created in 2014. Updated to 2015, also updated it to be "YCSB contributors" per discussion below.